src/screens/BiometricSettings/BiometricSettings.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useLayoutEffect, useState } from 'react';
+import { Switch, Text, View } from 'react-native';
+import * as LocalAuthentication from 'expo-local-authentication';
+import cloneDeep from 'lodash/cloneDeep';
+import backArrow from '../../navigators/utils/backArrow';
+import { updateBiometricUseStatus } from '../../store/account';
+import {
+  selectBiometricUseStatus,
+  selectLastLoggedUsername,
+} from '../../store/selectors/account';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { storeAsyncStorageBiometricUseStatus } from '../../util/asyncStorage';
+import { isBiometricSupported } from '../../util/biometric';
+import { colors } from '../../util/colors';
+import styles from './styles';
+
+const BiometricSettings = ({ navigation }) => {
+  const [isBiometricSupport, setIsBiometricSupport] = useState<boolean>(false);
+  const [usingStatus, setUsingStatus] = useState<boolean>(false);
+  const dispatch = useAppDispatch();
+  const lastLoggedUsername = useAppSelector(selectLastLoggedUsername);
+  const biometricUseStatus = useAppSelector(selectBiometricUseStatus);
+
+  useLayoutEffect(() => {
```

Comment:
Do we need this effect layout here? I assume we have arrow from navigation options in navigator

Reply:
I changed configuration. I dont need anymore navigation.setOption in this screen and deleted. 👍